examples/experimental/i3s-17-and-debug/obb-utils.js

```diff
@@ -0,0 +1,118 @@
+import {CompositeLayer, COORDINATE_SYSTEM, log} from '@deck.gl/core';
+import {SimpleMeshLayer} from '@deck.gl/mesh-layers';
+import SphereGeometry from './geometry/sphere-geometry';
+import CubeGeometry from './geometry/cube-geometry';
+import {OrientedBoundingBox} from '@math.gl/culling';
+
+const BG_OPACITY = 100;
+const GEOMETRY_STEP = 50;
+const SINGLE_DATA = [0];
+
+export default class ObbLayer extends CompositeLayer {
```

Comment:
A composite layer may be overly complicated. We are just rendering a single layer at the end, we could just use a SimpleMeshLayer and pass it the geometry and the data table with the instances?

Reply:
Will replace in next PR